src/datasource/plugin.json

```diff
@@ -1,12 +1,12 @@
 {
   "type": "datasource",
-  "name": "Zabbix",
-  "id": "alexanderzobnin-zabbix-datasource",
+  "name": "Iiris Zabbix Datasource",
+  "id": "iiris-zabbix-datasource",
   "metrics": true,
   "annotations": true,
   "backend": true,
   "alerting": true,
-  "executable": "gpx_zabbix-datasource",
+  "executable": "../iiris-zabbix-plugin",
```

Comment:
This seems a bit odd but I guess there's a reason for this.

Reply:
This is rather weird. I don't have this change on my dev machine and there are no changes to commit or push?

Reply:
Ah, I was on a wrong branch. I reverted this:
https://github.com/digiaiiris/grafana-zabbix/pull/57/commits/60d667f8a09c27e568e4afc9df63063877fac647